app/src/main/java/com/nervousfish/nervousfish/modules/database/GsonDatabaseAdapter.java

```diff
@@ -40,47 +34,27 @@
 @SuppressWarnings("PMD.TooManyMethods")
 public final class GsonDatabaseAdapter implements IDatabase {
 
-    private final static Type TYPE_CONTACT_LIST = new TypeToken<ArrayList<Contact>>(){}.getType();
-    private final static Type TYPE_PROFILE_LIST = new TypeToken<ArrayList<Profile>>(){}.getType();
+    private final static Type TYPE_CONTACT_LIST = new TypeToken<ArrayList<Contact>>() {
+    }.getType();
+    private final static Type TYPE_PROFILE_LIST = new TypeToken<ArrayList<Profile>>() {
+    }.getType();
```

Comment:
I'm sorry Eric, but no... I cannot I think :-8
If I change it back and do auto format in Intelllij, then it's changed back to this.